swagger.json

```diff
@@ -585,12 +602,6 @@
               "$ref" : "#/definitions/PageOfCertificateAuthority"
             }
           },
-          "400" : {
```

Comment:
I vaguely remember that the API threw 400 in some cases. Can you check out the backend code?

Reply:
I went over the REST controllers and backfilled a bunch of missing response codes.

I looked at this one specifically and it can respond with a 404 but not 400